### PR TITLE
Fix not ignoring missing for pluralization

### PR DESCRIPTION
**Pull request by pedrozath**
References #331

When you add a pattern for ignoring missing, it is currently not ignoring the pluralization for that pattern.
This commit fixes that, by skipping plural nodes that were ignored in the configuration for `ignore_missing`.

### CODE REVIEW

**Comment by yorsant** [comment]
References #331

File: lib/i18n/tasks/missing_keys.rb [hunk 1]
```diff
@@ -65,6 +65,7 @@ def missing_plural_forest(locales, _base = base_locale)
         plural_nodes data[locale] do |node|
           children = node.children
           present_keys = Set.new(children.map { |c| c.key.to_sym })
+          next if ignore_key?(node.key_names.first, :missing)
```
Comment (glebm) [inline lib/i18n/tasks/missing_keys.rb:68]:
```suggestion
          next if ignore_key?(node.full_key(root: false), :missing)
```

**Review by glebm — CHANGES_REQUESTED** [review]
Can you please also add or update the test for this?

**Comment by pedrozath** [comment]
@glebm thanks for the reviewing this! just updated the test :)

**Pull request merged**